Userland/Libraries/LibGfx/ICC/BinaryWriter.cpp

```diff
@@ -166,6 +166,62 @@ static ErrorOr<ByteBuffer> encode_signature(SignatureTagData const& tag_data)
     return bytes;
 }
 
+static ErrorOr<ByteBuffer> encode_text_description(TextDescriptionTagData const& tag_data)
+{
+    // ICC v2, 6.5.17 textDescriptionType
+    // All lengths include room for a trailing nul character.
+    // See also the many comments in TextDescriptionTagData::from_bytes().
+    u32 ascii_size = sizeof(u32) + tag_data.ascii_description().bytes().size() + 1;
+
+    // FIXME: Include tag_data.unicode_description() if it's set.
+    u32 unicode_size = 2 * sizeof(u32);
+
+    // FIXME: Include tag_data.macintosh_description() if it's set.
+    u32 macintosh_size = sizeof(u16) + sizeof(u8) + 67;
+
+    auto bytes = TRY(ByteBuffer::create_uninitialized(2 * sizeof(u32) + ascii_size + unicode_size + macintosh_size));
+    *bit_cast<BigEndian<u32>*>(bytes.data()) = (u32)TextDescriptionTagData::Type;
```

Comment:
Please use C++-style casts (i.e. `static_cast<u32>(..)`) or `to_underlying(..)` since you're using an enum.

Reply:
It's not an enum but a class type: https://github.com/SerenityOS/serenity/blob/master/Userland/Libraries/LibGfx/ICC/DistinctFourCC.h#L32

I can change it to `static_cast` if you want, but from what I can tell using C-style casts for calling class cast operators is pretty common in the codebase.

Reply:
changed it to static_cast

Reply:
Also added another commit that changes it in the other places in this file.

Reply:
> using C-style casts for calling class cast operators is pretty common in the codebase

Exactly, trying to prevent more coming in :sweat_smile: